pkg/combustion/templates/30-certificates.sh.tpl

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+
+cp ./{{ .CertificatesDir }}/* /etc/pki/trust/anchors/.
+update-ca-certificates
```

Comment:
We may want to think about making this output verbose so users can see this happening during boot up logs.

```suggestion
update-ca-certificates -v
```